html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label>
+    <br>
```

Comment:
while `br` tags will give you a line break, its better to use `div` elements to section off the form. Since they are block level, it will force things onto another line.

Reply:
<img width="946" alt="Screen Shot 2022-03-15 at 11 18 31 AM" src="https://user-images.githubusercontent.com/20695764/158445223-b4708618-c453-44cf-8a6c-0689e1b807df.png">